style.css

```diff
@@ -138,3 +132,95 @@ li a:hover {
     background: white;
     color: #535456;
 }
+.buttons{
+    font-size: 30px;
+    text-align: center;
+}
+.fa-bitbucket{
+    background: white;
+    color: #205081;
+}
+.fa-github-alt{
+    background: white;
+    color: black;
+}
+.logo{
+    font-size: 10px;
+    text-align: center;
+    text-decoration: none;
+    font-style: normal;
+    color: white;
+}
+.floating-box {
+    background-color: #b3c0cb;
+    float: left;
+    width: auto;
+    height: 150px;
+    margin: 10px;
+}
+.floating-box1 {
+    background-color: white;
+    display: inline-block;
+    float:left;
+    width: 5px;
+    height: 150px;
+    margin: 10px;
+}
+.floating-box2 {
+    background-color: #b3c0cb;
+    display: inline-block;
+    float:left;
+    text-align:justify;
+    width: 800px;
+    height: 150px;
+    margin: 10px;
+}
+ #scrlBtn{
+     position: sticky;
+     cursor:pointer;
+     background-color: #b3c0cb;
+     color: #546B7B;
+     border: #A4BCC9;
+     font-size: 30px;
+     text-align: center;
+
+ }
+ .imageContainer{
+     display:inline-block;
+     float:left;
+     width: 130px;
+     height: 150px;
+     margin: 50px;
+ }
+ .middleContainer {
+  transition: .5s ease;
```

Comment:
Is this transition and the transform line below actually working? It's hard to tell from just reading the code, but it looks like you're trying to do something that isn't fully implemented. Let me know if that's the case and you want help.